falcon/routing/compiled.py

```diff
@@ -294,11 +316,12 @@ def __init__(self, raw_segment,
         self.is_var = False
         self.is_complex = False
         self.var_name = None
+        self.var_pattern = None
 
         # NOTE(kgriffs): CompiledRouter.add_route validates field names,
         # so here we can just assume they are OK and use the simple
-        # _FIELD_REGEX to match them.
-        matches = list(_FIELD_REGEX.finditer(raw_segment))
+        # _FIELD_PATTERN to match them.
```

Comment:
I took the opportunity to normalize the terminology used in naming variables and such. "Pattern" is more commonly used in Falcon as opposed to "regex" or "expression".